src/main/resources/assets/js/application/services/tdpAuthService.js

```diff
@@ -0,0 +1,150 @@
+define(['angular','application/tdpInvestModule', 'ngCookies'], function(angular, tdpInvestModule) {
```

Comment:
We started the naming the filed with projectName prefix - this should be therefore named tdpInvestAuthService